OmniCore.Model/OmniCore.Model.Eros/ErosPodManager.cs

```diff
@@ -134,7 +134,7 @@ private ErosMessageExchangeParameters GetStandardParameters()
                 progress.ActionText = "Started new message exchange";
                 progress.Result.RequestTime = DateTimeOffset.UtcNow;
                 progress.Running = true;
-                var messageExchange = await MessageExchangeProvider.GetMessageExchange(messageExchangeParameters, Pod);
+                var messageExchange = await MessageExchangeProvider.GetMessageExchange(messageExchangeParameters, Pod, progress);
```

Comment:
see comment about design, this class is no more

Reply:
See e
Was the easiest way to implement the change for rebuilding the RileyLinkInstance in GetMessageExchange().

See my comments earlier on the same